src/js/stores/AppsStore.js

```diff
@@ -115,8 +115,15 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function calculateTotalResources(app) {
+  app.totalMem = app.mem * app.instances;
+  app.totalCpus = parseFloat((app.cpus * app.instances).toPrecision(2));
```

Comment:
It could make sense to increase the precision here to 4?
On my (sucessfull) tests to start 100.000 tasks on my machine, I set the CPU resource request to 0.0001, so this could be reflected here too. Not sure..